runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -111,6 +115,22 @@ private PythonExecutionResult runPythonProcess(String pythonPath, TempExecutionE
         }
     }
 
+    private String formatException(String exception, List<String> traceback) {
+        String formattedException = "";
+        if (traceback.size() > 0) {
+            formattedException = removeFileName(traceback.get(traceback.size() - 1));
+        } else {
```

Comment:
If you do the first change, you can also remove the else and just return 
`if(something){
   return a;
}
return b;`   This is called Early Return pattern